@sanity/vision is missing a dependency on sanity

**Describe the bug**

`@sanity/vision` imports from `sanity` but does not declare `sanity` as a dependency. It will fail to resolve in environments that symlink dependencies, for example pnpm with `enableGlobalVirtualStore`

**To Reproduce**

Steps to reproduce the behavior:

1. initialize a sanity project with pnpm
2. enable global virtual store
```
.npmrc
enable-global-virtual-store=true
```
4. Delete node_modules and reinstall to ensure you're using the global store
5. Run any code that imports sanity vision (for example, a config that uses the package)

You'll see several errors like the following:
```
Module not found: Can't resolve 'sanity'
  1 | import { EyeOpenIcon } from "@sanity/icons";
  2 | import { lazy } from "react";
> 3 | import { defineLocaleResourceBundle, definePlugin } from "sanity";
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  4 | import { route } from "sanity/router";
  5 | const visionLocaleNamespace = "vision", visionUsEnglishLocaleBundle = defineLocaleResourceBundle({
  6 |   locale: "en-US",


Module not found: Can't resolve 'sanity/router'
  2 | import { lazy } from "react";
  3 | import { defineLocaleResourceBundle, definePlugin } from "sanity";
> 4 | import { route } from "sanity/router";
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  5 | const visionLocaleNamespace = "vision", visionUsEnglishLocaleBundle = defineLocaleResourceBundle({
  6 |   locale: "en-US",
  7 |   namespace: visionLocaleNamespace,

```

**Which versions of Sanity are you using?**

```
@sanity/cli (global)  4.21.1 (up to date)
@sanity/assist         5.0.2 (up to date)
@sanity/code-input     6.0.3 (up to date)
@sanity/icons          3.7.4 (up to date)
@sanity/image-url      2.0.2 (up to date)
@sanity/vision        4.21.1 (up to date)
sanity                4.21.1 (up to date)
```

**What operating system are you using?**

Macos Sequoia Version 15.6.1 (24G90)

**Which versions of Node.js / npm are you running?**

```
> node -v && npm -v && pnpm -v
v24.11.1
11.6.2
10.25.0
```

**Workaround**

Luckily, pnpm provides a way to extend package dependencies:
```
# pnpm-workspace.yaml
packageExtensions:
  "@sanity/vision":
    dependencies:
      sanity: "*"
```

--- a/packages/@sanity/vision/package.json
+++ b/packages/@sanity/vision/package.json
@@ -102,6 +102,7 @@
   },
   "peerDependencies": {
     "react": "^18 || ^19.0.0",
+    "sanity": "workspace:^",
     "styled-components": "^6.1.15"
   }
 }

